modelforge/potential/models.py

```diff
@@ -482,19 +482,20 @@ def setup_potential(
                 f"Unsupported neighborlist strategy: {neighborlist_strategy}"
             )
 
-    model = Potential(
+    potential = Potential(
         core_network,
         neighborlist,
         postprocessing,
         jit=jit,
         jit_neighborlist=False if use_training_mode_neighborlist else True,
     )
-    model.eval()
-    return model
+    potential.eval()
+    return potential
 
 
 from openff.units import unit
 
+from modelforge.train.training import ModelTrainer
```

Comment:
Were you going to change the name of `ModelTrainer` to `PotentialTrainer`?

Reply:
That actually makes more sense, and I am changing it now.